src/renderer/type/v10-content-type-renderer.ts

```diff
@@ -0,0 +1,126 @@
+import { ContentTypeField } from 'contentful';
+import {
+  OptionalKind,
+  PropertySignatureStructure,
+  SourceFile,
+  TypeAliasDeclarationStructure,
+} from 'ts-morph';
+import { propertyImports } from '../../property-imports';
+import { renderTypeGeneric } from '../generic';
+import { CFContentType } from '../../types';
+import { BaseContentTypeRenderer } from './base-content-type-renderer';
+import { RenderContext } from './create-default-context';
+import { createV10Context } from './create-v10-context';
+
+export class V10ContentTypeRenderer extends BaseContentTypeRenderer {
+  public render(contentType: CFContentType, file: SourceFile): void {
+    const context = this.createContext();
+
+    this.addDefaultImports(context);
+    this.renderFieldsInterface(contentType, file, context);
+
+    file.addTypeAlias(this.renderEntrySkeleton(contentType, context));
+
+    file.addTypeAlias(this.renderEntry(contentType, context));
+
+    for (const structure of context.imports) {
+      file.addImportDeclaration(structure);
+    }
```

Comment:
Out of curiosity, have you tried to call 

```
file.organizeImports({
      ensureNewLineAtEndOfFile: true,
    });
 ```
 
 at the end to combine imports from `contentful`?

Reply:
I’ve added it now.